test/test.js

```diff
@@ -44,6 +44,31 @@ describe('auto-check element', function() {
       assert.isFalse(input.checkValidity())
     })
 
+    it('invalidates the input element on keypress', async function() {
+      const inputEvent = once(input, 'change')
+      triggerChange(input, 'hub')
+      await inputEvent
+      assert.isFalse(input.checkValidity())
+    })
+
+    it('validates the input element if there is no `src` attribute', async function() {
```

Comment:
The auto-check element should be inert and perform no behaviors if a required attribute, like `src`, is missing.